package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "leaderboard_joserm",
   "version": "1.0.0",
-  "description": "",
+  "description": "<a name=\"readme-top\"></a>",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
- [ ] Kindly consider adding the necessary `npm` script to successfully `run` and `build` your application. This will give anyone who clones your repo an easy start to running your application without any issues. Your script should look like below:
```
 "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack --config webpack.config.js --watch"
  }
```